dubbo-admin-backend/src/main/java/org/apache/dubbo/admin/config/ConfigCenter.java

```diff
@@ -43,7 +43,7 @@
     @Value("${dubbo.configcenter:}")
     private String configCenter;
 
-    @Value("${dubbo.registry.address:}")
+    @Value("${dubbo.configcenter:}")
     private String registryAddress;
 
     private static String globalConfigPath = "config/dubbo/dubbo.properties";
```

Comment:
both dubbo.configcenter and dubbo.registrey.address are necessary here, do not change

Reply:
if don't change this , can I change the  application.properties add a config ?
like this: dubbo.registry.address=zookeeper://127.0.0.1:2181